__tests__/package.json

```diff
@@ -1,28 +1,20 @@
 {
   "name": "eslint-config-upleveled-tests",
   "version": "1.0.0",
-  "main": "index.js",
-  "author": "Karl Horky <karl.horky@gmail.com>",
-  "license": "MIT",
   "private": true,
+  "license": "MIT",
+  "author": "Karl Horky <karl.horky@gmail.com>",
+  "main": "index.js",
```

Comment:
if these lines will be re-sorted, you can also change the author to whatever we normally write for UpLeveled and add a `contributors` array, like mentioned in the Stylelint config PR:

- https://github.com/upleveled/stylelint-config-upleveled/pull/3

Reply:
https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/2bf8840f52e0b13dd127f65d00fc147a4c5c5452
https://github.com/upleveled/eslint-config-upleveled/pull/220/commits/690a06c9694734491dca25dcd2d22f6b39f76e45